dev/com.ibm.ws.beanvalidation.v20.cdi/src/com/ibm/ws/beanvalidation/v20/cdi/internal/ValidationReleasableFactoryImpl.java

```diff
@@ -45,7 +53,7 @@
 /**
  * An implementation that is CDI aware.
  */
-@Component(configurationPolicy = ConfigurationPolicy.IGNORE,
+@Component(configurationPolicy = ConfigurationPolicy.OPTIONAL,
```

Comment:
Why are we using OPTIONAL here as opposed to REQUIRED?  We should use REQUIRED if possible.